tests/app/authentication/test_jwt_decoder_using_rfc_example.py

```diff
@@ -87,7 +87,7 @@ def test_decrypt(self):
         cipher_text = decoder._base64_decode(tokens[3])
         tag = decoder._base64_decode(tokens[4])
 
-        decrypted_token = decoder._decrypt_cipher_text(cipher_text, iv, cek, tag, jwe_protected_header)
+        decrypted_token = decoder._decrypt_cipher_text(cipher_text, bytes(iv), bytes(cek), bytes(tag), jwe_protected_header)
```

Comment:
Could be clearer to put the bytes() wrapping around the cek/iv when they are declared above rather than here. Also the tag doesn't need this conversion, but belt and braces you could wrap the decoder.__base64_decode call rather than here as well?